specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.Communication/AzureCommunicationServices.tsp

```diff
@@ -1160,6 +1160,28 @@ model AcsMessageReceivedEventData extends AcsMessageEventData {
   interactive: AcsMessageInteractiveContent;
 }
 
+/** Schema of the Data property of an EventGridEvent for a Microsoft.Communication.AdvancedMessageAnalysisCompleted event. */
+interface AcsMessageAnalysisCompletedEventData extends AcsMessageEventData {
+  /** The original message received. */
+  originalMessage: string;
+
+  /** The analysed message channel kind. */
+  channelKind: "whatsapp";
```

Comment:
In the swagger file (in this PR), the data type is mentioned as `AcsMessageChannelKind`. Which could be 'whatsapp` or any string. Here it is mentioned just as 'whatsapp'. This should be changed.